build.gradle

```diff
@@ -1027,6 +1027,27 @@ task product(type: Zip) {
       from("${clusterProject.projectDir}/benchmark")
       into "${snappyProductDir}/benchmark"
     }
+
+    if (rootProject.hasProperty('R.enable') &&
+        rootProject.property("R.enable").toString().equalsIgnoreCase("true")) {
```

Comment:
Second check is not required. Our convention has been just the check for property so user will run: ./gradlew product -PR.enable